andela/Titan-Devs
# #163518674 Configure Eslint

#### What does this PR do?
Configures eslint for the project
#### Description of Task to be completed?
* Install eslint-airbnb-base and it's peer-dependencies
* Create .eslintr.json file for eslint configurations
#### How should this be manually tested?
* Clone this GitHub repo, and `cd` the root directory.
* Run `yarn ` to install all dependencies.
* Install *Eslint* and *Prettier* extensions in your VS code. 
* Add this line in your VS code settings `"editor.formatOnSave": true` for formatting on save.
* Create a random javascript file (it has to be deleted after), and write some lines of code. 
* Run `./node_modules/.bin/eslint name_of_random_file.js` to see potential errors eslint detected. 
#### What are the relevant pivotal tracker stories?
#163518674
